scripts/gen_stub.py

```diff
@@ -0,0 +1,45 @@
+from inspect import Parameter, Signature
+from typing import ForwardRef, Type
+
+from great_expectations.datasource.fluent import Datasource, PandasFilesystemDatasource
+
```

Comment:
Not necessary now but would be nice to have a top level docstring explaining what this is